tools/release/lib/release.py

```diff
@@ -15,99 +23,180 @@ def create_release(repository, tag, notes):
             tag=tag, name=tag, message=notes, draft=False, prerelease=False
         )
 
-    except Exception as e:
-        raise Exception(f"create_release: Error creating release/tag {tag}: {str(e)}")
+    except Exception as exp:
+        raise Exception(
+            f"create_release: Error creating release/tag {tag}: {str(exp)}"
+        ) from exp
+
 
+def get_sorted_merged_pulls(
+    pulls: List[PullRequest], last_release: Optional[GitRelease]
+) -> List[PullRequest]:
+    # Get merged pulls after last release
+    if not last_release:
+        return sorted(
+            (pull for pull in pulls if pull.merged and not pull.title.startswith("chore: release")),
```

Comment:
We need to also limit the author of this pull request title. If we choose to not use the script for any particular reason, this line would be wrong.

Reply:
So let me see if i get it, It shouldn't be in here? :
https://github.com/ada-url/ada/blob/f393ed6e9fd32d289378949f61cc62e23ca268d5/.github/workflows/release_create.yml#L20

Or you mean really in release.py?

Reply:
in `release.py` as well

Reply:
Done! (for release.py and release_create.yml)